bin/datatool/src/main.rs

```diff
@@ -1,231 +1,36 @@
-use std::{
-    fs,
-    path::{Path, PathBuf},
-};
+//! Command line tool for generating test data for Strata.
+//!
+//! # Warning
+//!
+//! This tool is intended for use in testing and development only. It generates
+//! keys and other data that should not be used in production.
+mod args;
+mod utils;
 
-use argh::FromArgs;
-use bitcoin::{
-    base58,
-    bip32::{ChildNumber, DerivationPath, Xpriv, Xpub},
-    secp256k1::SECP256K1,
-    Network,
-};
-use rand::{rngs::OsRng, CryptoRng, RngCore};
-use strata_primitives::{
-    block_credential,
-    buf::Buf32,
-    operator::OperatorPubkeys,
-    params::{ProofPublishMode, RollupParams},
-    vk::RollupVerifyingKey,
-};
-use strata_sp1_guest_builder::GUEST_CHECKPOINT_VK_HASH_STR;
+use std::path::PathBuf;
 
-// TODO move some of these into a keyderiv crate
-const DERIV_BASE_IDX: u32 = 56;
-const DERIV_SEQ_IDX: u32 = 10;
-const DERIV_OP_IDX: u32 = 20;
-const DERIV_OP_SIGNING_IDX: u32 = 100;
-const DERIV_OP_WALLET_IDX: u32 = 101;
-const SEQKEY_ENVVAR: &str = "STRATA_SEQ_KEY";
-const OPKEY_ENVVAR: &str = "STRATA_OP_KEY";
-const DEFAULT_NETWORK: Network = Network::Signet;
+use args::CmdContext;
+use rand::rngs::OsRng;
+use utils::{exec_subc, resolve_network};
 
 /// List of keys that are used in examples that we don't want people to actually
-/// use.  If any of these are present in a command invocation we abort immediately.
+/// use.
+///
+/// If any of these are present in a command invocation we abort immediately.
 const KEY_BLACKLIST: &[&str] = &[
```

Comment:
Removed this since we have a warning already in https://github.com/alpenlabs/strata/blob/f87f46979443e2f00668abe110d538b0aa1d91f4/bin/datatool/src/main.rs#L3-L6

Done in c5231a04983152267f97f0a7480abe309c3f9e2a